cmd/example/docs/generated/bars.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html><html lang=en><head><title>Bars</title><meta name='generator' content='github.com/lonnblad/go-service-doc'><link rel="stylesheet" href="/docs/service/markdown.css"></head><body class="markdown-body"><div class="flex-container"><div class="menu-container"><div  class=menu-header><h1>Bars</h1><form class=menu-search action="/bars/search" method="get"><input type="text" placeholder="Search.." name="q" value="" onfocus="var temp_value=this.value; this.value=''; this.value=temp_value" autofocus /><button type="submit">Search</button></form></div><ul><li><a href="/bars#bars">Bars</a></li><li><a href="/bars/donkey-bar#donkey">Donkey Bar</a><ul><li><a href="/bars/donkey-bar#code_examples">Code Examples</a></li></ul></li><li><a href="/bars/monkey-bar#monkey">Monkey Bar</a><ul><li><a href="/bars/monkey-bar#code_examples">Code Examples</a></li></ul></li></ul></div><div class="doc-container"><h1 id="bars">Bars</h1><p>| Bar Name   | Animal Family |<br />
```

Comment:
The address to markdown.css is `/docs/service/markdown.css`, but it should be `/bar/markdown.css` right?